kwok/kwok_script_runner.go

```diff
@@ -0,0 +1,86 @@
+package kwok
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+)
+
+// GetKubeconfig get the full path to the kwok directory's
+// kubeconfig file, no matter where the functions is called from.
+func getKubeconfig() (string, error) {
+	_, filename, _, ok := runtime.Caller(0)
+	if !ok {
+		return "", fmt.Errorf("failed to find the path of the current script")
+	}
+	dirPath := filepath.Dir(filename)
+	kubeconfigPath := filepath.Join(dirPath, "kubeconfig")
+	return kubeconfigPath, nil
+}
+
+// setKubeconfig set the kubeconfig using an environment variable.
+// It eithers get the full path to the kubeconfig, or set it to the
+// one in the kwok directory.
+func setKubeconfig(kubeconfigPath string) error {
```

Comment:
do we really need this function and this "if", do we ever call it without an empty string?

Reply:
As of now, we can go on without it